--- application-menu.coffee.orig	2015-09-22 05:48:42.000000000 -0700
+++ application-menu.coffee	2015-09-23 06:13:34.000000000 -0700
@@ -118,7 +118,7 @@
   # Returns an Array of menu item Objects.
   getDefaultTemplate: ->
     [
-      label: "Atom"
+      label: "Particle Dev"
       submenu: [
           {label: "Check for Update", metadata: {autoUpdate: true}}
           {label: 'Reload', accelerator: 'Command+R', click: => @focusedWindow()?.reload()}
