--- atom-application.coffee.orig	2017-04-06 17:26:42.000000000 +0100
+++ atom-application.coffee	2017-04-06 17:28:04.000000000 +0100
@@ -32,9 +32,9 @@
   @open: (options) ->
     unless options.socketPath?
       if process.platform is 'win32'
-        options.socketPath = "\\\\.\\pipe\\atom-#{options.version}-sock"
+        options.socketPath = "\\\\.\\pipe\\particledev-#{options.version}-sock"
       else
-        options.socketPath = path.join(os.tmpdir(), "atom-#{options.version}-#{process.env.USER}.sock")
+        options.socketPath = path.join(os.tmpdir(), "particledev-#{options.version}-#{process.env.USER}.sock")
 
     createAtomApplication = -> new AtomApplication(options)
 
