--- atom-application.coffee.orig	2016-02-12 05:17:59.000000000 -0800
+++ atom-application.coffee	2016-02-12 05:30:45.000000000 -0800
@@ -34,9 +34,9 @@
   @open: (options) ->
     unless options.socketPath?
       if process.platform is 'win32'
-        options.socketPath = '\\\\.\\pipe\\atom-sock'
+        options.socketPath = '\\\\.\\pipe\\particledev-sock'
       else
-        options.socketPath = path.join(os.tmpdir(), "atom-#{options.version}-#{process.env.USER}.sock")
+        options.socketPath = path.join(os.tmpdir(), "particledev-#{options.version}-#{process.env.USER}.sock")

     createAtomApplication = -> new AtomApplication(options)
