--- atom-window.coffee.orig	2017-09-21 13:48:17.000000000 +0200
+++ atom-window.coffee	2017-09-21 13:48:58.000000000 +0200
@@ -25,7 +25,7 @@
 
     options =
       show: false
-      title: 'Atom'
+      title: 'Particle Desktop IDE'
       webPreferences:
         # Prevent specs from throttling when the window is in the background:
         # this should result in faster CI builds, and an improvement in the
