--- auto-update-manager.coffee.orig	2017-02-02 18:02:30.000000000 +0000
+++ auto-update-manager.coffee	2017-02-02 18:05:22.000000000 +0000
@@ -18,7 +18,7 @@
   constructor: (@version, @testMode, resourcePath, @config) ->
     @state = IdleState
     @iconPath = path.resolve(__dirname, '..', '..', 'resources', 'atom.png')
-    @feedUrl = "https://atom.io/api/updates?version=#{@version}"
+    @feedUrl = "https://updates.particle.io?version=#{@version}"
     process.nextTick => @setupAutoUpdater()
 
   setupAutoUpdater: ->
