--- code-sign-on-mac.js.orig	2017-07-19 06:58:27.000000000 -0500
+++ code-sign-on-mac.js	2017-07-19 07:04:44.000000000 -0500
@@ -49,7 +49,7 @@
     spawnSync('codesign', [
       '--deep', '--force', '--verbose',
       '--keychain', process.env.ATOM_MAC_CODE_SIGNING_KEYCHAIN,
-      '--sign', 'Developer ID Application: GitHub', packagedAppPath
+      '--sign', 'Developer ID Application: Spark Labs, Inc.', packagedAppPath
     ], {stdio: 'inherit'})
   } finally {
     if (!process.env.ATOM_MAC_CODE_SIGNING_CERT_PATH) {
