--- codesign-task.coffee.orig	2017-02-02 17:49:50.000000000 +0000
+++ codesign-task.coffee	2017-02-02 17:56:11.000000000 +0000
@@ -12,7 +12,7 @@
     unlockKeychain (error) ->
       return done(error) if error?
 
-      args = ['--deep', '--force', '--verbose', '--sign', 'Developer ID Application: GitHub', grunt.config.get('atom.shellAppDir')]
+      args = ['--deep', '--force', '--verbose', '--sign', 'Developer ID Application: Spark Labs, Inc.', grunt.config.get('atom.shellAppDir')]
       spawn {cmd: 'codesign', args: args}, (error) -> done(error)
 
   unlockKeychain = (callback) ->
@@ -33,7 +33,7 @@
 
   grunt.registerTask 'codesign:installer', 'CodeSign Windows installer (AtomSetup.exe)', ->
     done = @async()
-    atomSetupExePath = path.resolve(grunt.config.get('atom.buildDir'), 'installer', 'AtomSetup.exe')
+    atomSetupExePath = path.resolve(grunt.config.get('atom.buildDir'), 'installer', 'ParticleDevSetup.exe')
     signWindowsExecutable atomSetupExePath, (error) -> done(error)
 
   grunt.registerTask 'codesign:installer-deferred', 'Obtain cert and configure installer to perform CodeSign', ->
