--- crash-reporter-start.js.orig	2017-09-21 13:39:11.000000000 +0200
+++ crash-reporter-start.js	2017-09-21 13:46:42.000000000 +0200
@@ -1,9 +1,9 @@
 module.exports = function (extra) {
   const {crashReporter} = require('electron')
   crashReporter.start({
-    productName: 'Atom',
-    companyName: 'GitHub',
-    submitURL: 'https://crashreporter.atom.io',
+    productName: 'ParticleDev',
+    companyName: 'Particle',
+    submitURL: 'https://particle-dev-breakpad.herokuapp.com/crash_upload',
     uploadToServer: false,
     extra: extra
   })
