--- create-windows-installer.js.orig	2017-09-21 13:39:11.000000000 +0200
+++ create-windows-installer.js	2017-09-21 13:44:15.000000000 +0200
@@ -11,13 +11,16 @@
   const archSuffix = process.arch === 'ia32' ? '' : '-' + process.arch
   const options = {
     appDirectory: packagedAppPath,
-    authors: 'GitHub Inc.',
-    iconUrl: `https://raw.githubusercontent.com/atom/atom/master/resources/app-icons/${CONFIG.channel}/atom.ico`,
+    authors: 'Particle Industries.',
+    iconUrl: 'https://raw.githubusercontent.com/spark/particle-dev-app/master/build/resources/particle-dev.ico',
     loadingGif: path.join(CONFIG.repositoryRootPath, 'resources', 'win', 'loading.gif'),
     outputDirectory: CONFIG.buildOutputPath,
     noMsi: true,
-    remoteReleases: `https://atom.io/api/updates${archSuffix}?version=${CONFIG.appMetadata.version}`,
-    setupIcon: path.join(CONFIG.repositoryRootPath, 'resources', 'app-icons', CONFIG.channel, 'atom.ico')
+    remoteReleases: `https://updates.particle.io/updates${archSuffix}?version=${CONFIG.appMetadata.version}`,
+    setupIcon: path.join(CONFIG.repositoryRootPath, 'resources', 'app-icons', 'stable', 'atom.ico'),
+    productName: 'ParticleDev',
+    name: 'particledev',
+    exe: 'particledev.exe'
   }
 
   const cleanUp = () => {
