--- darwin.cson.orig	2016-02-11 07:39:59.000000000 -0800
+++ darwin.cson	2016-02-11 08:05:56.000000000 -0800
@@ -1,8 +1,8 @@
 'menu': [
   {
-    label: 'Atom'
+    label: 'Particle Dev'
     submenu: [
-      { label: 'About Atom', command: 'application:about' }
+      { label: 'About Particle Dev', command: 'application:about' }
       { label: 'View License', command: 'application:open-license' }
       { label: 'VERSION', enabled: false }
       { label: 'Restart and Install Update', command: 'application:install-update', visible: false}
@@ -21,7 +21,7 @@
       { type: 'separator' }
       { label: 'Services', role: 'services', submenu: [] }
       { type: 'separator' }
-      { label: 'Hide Atom', command: 'application:hide' }
+      { label: 'Hide Particle Dev', command: 'application:hide' }
       { label: 'Hide Others', command: 'application:hide-other-applications' }
       { label: 'Show All', command: 'application:unhide-all-applications' }
       { type: 'separator' }
@@ -138,7 +138,6 @@
         submenu: [
           { label: 'Open In Dev Mode…', command: 'application:open-dev' }
           { label: 'Reload Window', command: 'window:reload' }
-          { label: 'Run Package Specs', command: 'window:run-package-specs' }
           { label: 'Toggle Developer Tools', command: 'window:toggle-dev-tools' }
         ]
       }
