--- Gruntfile.coffee.orig	2016-02-11 09:38:16.000000000 -0800
+++ Gruntfile.coffee	2016-02-11 09:46:11.000000000 -0800
@@ -53,7 +53,7 @@
   channel ?= 'dev'

   metadata = packageJson
-  appName = packageJson.productName
+  appName = if process.platform is 'darwin' then 'Particle Dev' else 'ParticleDev'
   appFileName = packageJson.name
   apmFileName = 'apm'

@@ -266,11 +266,14 @@
       installer:
         appDirectory: shellAppDir
         outputDirectory: path.join(buildDir, 'installer')
-        authors: 'GitHub Inc.'
+        authors: 'Particle Industries, Inc.'
         loadingGif: path.resolve(__dirname, '..', 'resources', 'win', 'loading.gif')
-        iconUrl: "https://raw.githubusercontent.com/atom/atom/master/resources/app-icons/#{channel}/atom.ico"
+        iconUrl: "https://raw.githubusercontent.com/spark/spark-dev/master/build/resources/sparkdev.ico"
         setupIcon: path.resolve(__dirname, '..', 'resources', 'app-icons', channel, 'atom.ico')
-        remoteReleases: "https://atom.io/api/updates?version=#{metadata.version}"
+        remoteReleases: "https://github.com/spark/particle-dev-app"
+        productName: 'ParticleDev'
+        name: 'particle-dev'
+        exe: 'atom.exe'

     shell:
       'kill-atom':
