--- initialize-application-window.coffee.orig	2017-07-21 11:52:46.000000000 -0500
+++ initialize-application-window.coffee	2017-07-24 06:12:16.000000000 -0500
@@ -66,6 +66,22 @@
   enablePersistence: true
 })
 
+if global.isGeneratingSnapshot
+  # Remove injected packages from snapshot
+  delete global.atom.packages.packagesCache['switch-header-source']
+  delete global.atom.packages.packagesCache['resize-panes']
+  delete global.atom.packages.packagesCache['maximize-panes']
+  delete global.atom.packages.packagesCache['swap-panes']
+  delete global.atom.packages.packagesCache['tool-bar']
+  delete global.atom.packages.packagesCache['console-panel']
+  delete global.atom.packages.packagesCache['particle-dev']
+  delete global.atom.packages.packagesCache['particle-dev-release-notes']
+  delete global.atom.packages.packagesCache['particle-dev-exception-reporting']
+  delete global.atom.packages.packagesCache['particle-dev-cloud-functions']
+  delete global.atom.packages.packagesCache['particle-dev-cloud-variables']
+  delete global.atom.packages.packagesCache['particle-dev-profiles']
+  delete global.atom.packages.packagesCache['particle-dev-libraries']
+
 global.atom.preloadPackages()
 
 # Like sands through the hourglass, so are the days of our lives.
