--- license-overrides.coffee.orig	2017-04-06 17:29:40.000000000 +0100
+++ license-overrides.coffee	2017-04-07 11:38:03.000000000 +0100
@@ -123,7 +123,7 @@
     (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
     THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
     """
-  'json-schema@0.2.2':
+  'json-schema@0.2.3':
     repository: 'https://github.com/kriszyp/json-schema'
     license: 'BSD'
     source: 'README links to https://github.com/dojo/dojo/blob/8b6a5e4c42f9cf777dd39eaae8b188e0ebb59a4c/LICENSE'
@@ -352,3 +352,6 @@
     You can copy, modify, distribute and perform the work, even for commercial
     purposes, all without asking permission.
     """
+  'tweetnacl@0.14.1':
+    license: 'Public Domain'
+    source: 'https://raw.githubusercontent.com/dchest/tweetnacl-js/master/COPYING.txt'
