--- license-overrides.coffee.orig	2016-03-10 16:57:54.000000000 +0000
+++ license-overrides.coffee	2016-03-10 17:01:20.000000000 +0000
@@ -82,3 +82,15 @@
   'core-js@0.4.10':
     license: 'MIT'
     source: 'http://rock.mit-license.org linked in source files and bower.json says MIT'
+  'rc@1.1.2':
+    license: 'MIT'
+    source: 'LICENSE.MIT'
+  'spdx-exceptions@1.0.3':
+    license: 'Unlicense'
+    source: 'Actually CC-BY-3.0 but it fails in legal-eagle'
+  'json-schema@0.2.2':
+    license: 'BSD'
+    source: 'http://trac.dojotoolkit.org/browser/dojo/trunk/LICENSE#L13'
+  'tweetnacl@0.14.1':
+    license: 'Public Domain'
+    source: 'https://raw.githubusercontent.com/dchest/tweetnacl-js/master/COPYING.txt'
