--- main.js.orig	2017-04-06 17:19:33.000000000 +0100
+++ main.js	2017-04-06 17:21:22.000000000 +0100
@@ -38,7 +38,7 @@
   }
 
   // NB: This prevents Win10 from showing dupe items in the taskbar
-  app.setAppUserModelId('com.squirrel.atom.atom')
+  app.setAppUserModelId('com.squirrel.particle.particledev')
 
   function addPathToOpen (event, pathToOpen) {
     event.preventDefault()
@@ -95,7 +95,7 @@
     return
   }
 
-  let atomHome = path.join(app.getPath('home'), '.atom')
+  let atomHome = path.join(app.getPath('home'), '.particledev')
   const AtomPortable = require('./atom-portable')
 
   if (setPortable && !AtomPortable.isPortableInstall(process.platform, process.env.ATOM_HOME, atomHome)) {
