--- main.coffee.orig	2016-05-10 13:01:58.000000000 +0100
+++ main.coffee	2016-05-10 14:39:54.000000000 +0100
@@ -19,7 +19,7 @@
   return if handleStartupEventWithSquirrel()

   # NB: This prevents Win10 from showing dupe items in the taskbar
-  app.setAppUserModelId('com.squirrel.atom.atom')
+  app.setAppUserModelId('com.squirrel.particle.particledev')

   addPathToOpen = (event, pathToOpen) ->
     event.preventDefault()
@@ -60,12 +60,12 @@
   SquirrelUpdate.handleStartupEvent(app, squirrelCommand)

 setupCrashReporter = ->
-  crashReporter.start(productName: 'Atom', companyName: 'GitHub', submitURL: 'http://54.249.141.255:1127/post')
+  crashReporter.start(productName: 'ParticleDev', companyName: 'Particle', submitURL: 'https://particle-dev-breakpad.herokuapp.com/crash_upload')

 setupAtomHome = ({setPortable}) ->
   return if process.env.ATOM_HOME

-  atomHome = path.join(app.getPath('home'), '.atom')
+  atomHome = path.join(app.getPath('home'), '.particledev')
   AtomPortable = require './atom-portable'

   if setPortable and not AtomPortable.isPortableInstall(process.platform, process.env.ATOM_HOME, atomHome)
