--- publish-build-task.coffee.orig	2017-04-05 14:23:49.000000000 +0100
+++ publish-build-task.coffee	2017-04-05 14:24:50.000000000 +0100
@@ -74,13 +74,11 @@
   switch process.platform
     when 'darwin'
       [
-        {assetName: 'atom-mac.zip', sourcePath: appName}
-        {assetName: 'atom-mac-symbols.zip', sourcePath: 'Atom.breakpad.syms'}
-        {assetName: 'atom-api.json', sourcePath: 'atom-api.json'}
+        {assetName: "particle-dev-mac-#{version}.zip", sourcePath: 'Particle Dev.app'}
       ]
     when 'win32'
-      assets = [{assetName: 'atom-windows.zip', sourcePath: appName}]
-      for squirrelAsset in ['AtomSetup.exe', 'AtomSetup.msi', 'RELEASES', "atom-#{version}-full.nupkg", "atom-#{version}-delta.nupkg"]
+      assets = [{assetName: "particle-dev-windows-#{version}.zip", sourcePath: 'ParticleDev'}]
+      for squirrelAsset in ['ParticleDevSetup.exe', 'ParticleDevSetup.msi', 'RELEASES', "particledev-#{version}-full.nupkg", "particledev-#{version}-delta.nupkg"]
         cp path.join(buildDir, 'installer', squirrelAsset), path.join(buildDir, squirrelAsset)
         assets.push({assetName: squirrelAsset, sourcePath: assetName})
       assets
@@ -102,7 +100,7 @@
           arch = 'i386'
         else
           arch = 'x86_64'
-        assetName = "atom.#{arch}.rpm"
+        assetName = "particle-dev.#{arch}.rpm"
 
       cp sourcePath, path.join(buildDir, assetName)
       assets = [{assetName, sourcePath}]
