--- publish-build-task.coffee.orig	2016-02-11 08:45:44.000000000 -0800
+++ publish-build-task.coffee	2016-02-11 09:24:43.000000000 -0800
@@ -73,13 +73,11 @@
   switch process.platform
     when 'darwin'
       [
-        {assetName: 'atom-mac.zip', sourcePath: appName}
-        {assetName: 'atom-mac-symbols.zip', sourcePath: 'Atom.breakpad.syms'}
-        {assetName: 'atom-api.json', sourcePath: 'atom-api.json'}
+        {assetName: "particle-dev-mac-#{version}.zip", sourcePath: 'Particle Dev.app'}
       ]
     when 'win32'
-      assets = [{assetName: 'atom-windows.zip', sourcePath: appName}]
-      for squirrelAsset in ['AtomSetup.exe', 'RELEASES', "atom-#{version}-full.nupkg", "atom-#{version}-delta.nupkg"]
+      assets = [{assetName: "particle-dev-windows-#{version}.zip", sourcePath: 'ParticleDev'}]
+      for squirrelAsset in ['ParticleDevSetup.exe', 'RELEASES', "particledev-#{version}-full.nupkg", "particledev-#{version}-delta.nupkg"]
         cp path.join(buildDir, 'installer', squirrelAsset), path.join(buildDir, squirrelAsset)
         assets.push({assetName: squirrelAsset, sourcePath: assetName})
       assets
@@ -101,7 +99,7 @@
           arch = 'i386'
         else
           arch = 'x86_64'
-        assetName = "atom.#{arch}.rpm"
+        assetName = "particle-dev.#{arch}.rpm"

       cp sourcePath, path.join(buildDir, assetName)

@@ -132,10 +130,10 @@
   async.parallel(tasks, callback)

 getAtomDraftRelease = (isPrerelease, branchName, callback) ->
-  atomRepo = new GitHub({repo: 'atom/atom', token})
+  atomRepo = new GitHub({repo: 'spark/particle-dev-app', token})
   atomRepo.getReleases {prerelease: isPrerelease}, (error, releases=[]) ->
     if error?
-      logError('Fetching atom/atom releases failed', error, releases)
+      logError('Fetching spark/particle-dev-app releases failed', error, releases)
       callback(error)
     else
       [firstDraft] = releases.filter ({draft}) -> draft
@@ -158,7 +156,7 @@
 createAtomDraftRelease = (isPrerelease, branchName, callback) ->
   {version} = require('../../package.json')
   options =
-    uri: 'https://api.github.com/repos/atom/atom/releases'
+    uri: 'https://api.github.com/repos/spark/particle-dev-app/releases'
     method: 'POST'
     headers: defaultHeaders
     json:
@@ -175,7 +173,7 @@

   request options, (error, response, body='') ->
     if error? or response.statusCode isnt 201
-      logError("Creating atom/atom draft release failed", error, body)
+      logError("Creating spark/particle-dev-app draft release failed", error, body)
       callback(error ? new Error(response.statusCode))
     else
       callback(null, body)
