--- workspace.js.orig	2017-07-19 06:50:11.000000000 -0500
+++ workspace.js	2017-07-19 06:52:31.000000000 -0500
@@ -542,7 +542,7 @@
   // open.
   updateWindowTitle () {
     let itemPath, itemTitle, projectPath, representedPath
-    const appName = 'Atom'
+    const appName = 'Particle Desktop IDE'
     const left = this.project.getPaths()
     const projectPaths = left != null ? left : []
     const item = this.getActivePaneItem()
